pythonprep_assignment.py

```diff
@@ -12,44 +12,32 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+def define_objects():
 
-# TODO:  Add code here to read the file into a string
-def read_file_into_string(filename):
-    """Returns a string with the file's contents
+    # TODO:  Add code here to define a list containing the names of the
+    # months. Please use three-character month names, like "Jan" for January.
+    # Please call it "Months".
 
-    Args:
-        filename(string): name of the file to open
-    """
+    # TODO: Add code here to define a dictionary containing the days of the
+    # month for your current year. Please use three-character month names, like
+    # "Jan" for "January". Please call it "Days".
 
-    # TODO: Add code to read the file into a string
+    return Months, Days
 
-    return data
 
+def compute_30_day_months(Months, Days):
 
-# TODO:  Add code here to break the string into a list of numbers
-def break_file_data_into_list_of_numbers(string_data):
-    """Returns a list with the numbers in the file
+    num_30 = 0
+    for Month in Months:
```

Comment:
`month`, `days` etc should be lower case [pep8](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names)

Reply:
OK. Making these changes.